docs/architecture/devops-for-aspnet-developers/deploying-to-app-service.md

```diff
@@ -0,0 +1,245 @@
+
+4. In a browser, navigate to the *Web app URL* and note the app has been built and deployed. Additional changes can be committed to the local Git repository with `git commit`. These changes are pushed to Azure with the preceding `git push` command.
+
+## Deployment with Visual Studio
+
+> *Note: This section applies to Windows only. Linux and macOS users should make the change described in step 2 below. Save the file, and commit the change to the local repository with `git commit`. Finally, push the change with `git push`, as in the first section.*
+
+The app has already been deployed from the command shell. Let's use Visual Studio's integrated tools to deploy an update to the app. Behind the scenes, Visual Studio accomplishes the same thing as the command line tooling, but within Visual Studio's familiar UI.
+
+1. Open *SimpleFeedReader.sln* in Visual Studio.
+2. In Solution Explorer, open *Pages\Index.cshtml*. Change `<h2>Simple Feed Reader</h2>` to `<h2>Simple Feed Reader - V2</h2>`.
+3. Press **Ctrl**+**Shift**+**B** to build the app.
+4. In Solution Explorer, right-click on the project and click **Publish**.
+
+    ![Screenshot showing Right-click, Publish](./media/deploying-to-app-service/publish.png)
+5. Visual Studio can create a new App Service resource, but this update will be published over the existing deployment. In the **Pick a publish target** dialog, select **App Service** from the list on the left, and then select **Select Existing**. Click **Publish**.
+6. In the **App Service** dialog, confirm that the Microsoft or Organizational account used to create your Azure subscription is displayed in the upper right. If it's not, click the drop-down and add it.
+7. Confirm that the correct Azure **Subscription** is selected. For **View**, select **Resource Group**. Expand the **AzureTutorial** resource group and then select the existing web app. Click **OK**.
+
+    ![Screenshot showing Publish App Service dialog](./media/deploying-to-app-service/publish-dialog.png)
+
+Visual Studio builds and deploys the app to Azure. Browse to the web app URL. Validate that the `<h2>` element modification is live.
+
+![The app with the changed title](./media/deploying-to-app-service/app-v2.png)
+
+## Deployment slots
+
+Deployment slots support the staging of changes without impacting the app running in production. Once the staged version of the app is validated by a quality assurance team, the production and staging slots can be swapped. The app in staging is promoted to production in this manner. The following steps create a staging slot, deploy some changes to it, and swap the staging slot with production after verification.
+
+1. Sign in to the [Azure Cloud Shell](https://shell.azure.com/bash), if not already signed in.
+2. Create the staging slot.
+
+    a. Create a deployment slot with the name *staging*.
+
+    ```azurecli
+    az webapp deployment slot create --name $webappname --resource-group AzureTutorial --slot staging
+    ```
```

Comment:
It may look nicer if you replace `a.` with `1.` and indent the command. (here and below)

Reply:
I think we can leave this as is as this looks good to me.

![image](https://user-images.githubusercontent.com/13281246/114853291-441f7400-9e01-11eb-9bf1-e9f6d92fef94.png)